Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,453 @@
+                                              DeviceEventCategory == 32768, "Modem Debug (32768)",
+                                              DeviceEventCategory == 65536, "VPN Tunnel Status (65536)",
+                                              DeviceEventCategory == 131072, "IEEE 802.11 Management (131072)",
+                                              DeviceEventCategory == 262144, "Connection Opened (262144)",
+                                              DeviceEventCategory == 524288, "System Environment (524288)",
+                                              DeviceEventCategory == 1048576, "Expanded - VoIP Activity (1048576)",
+                                              DeviceEventCategory == 2097152, "Expanded - WLAN IDS Activity (2097152)",
+                                              DeviceEventCategory == 4194304, "Expanded - SonicPoint Activity (4194304)",
+                                              DeviceEventCategory == 8388608, "Expanded - Unified Policy Engine (8388608)",
+                                              "Legacy Category Not Mapped"
+                                              )
+  | extend sosIPSPriority = case(ipspri == 1, "High (1)",
+                                              ipspri == 2, "Medium (2)",
+                                              ipspri == 3, "Low (3)",
+                                              ""
+                                              )
+  | extend sosAntiSpywarePriority = case(spypri == 1, "High (1)",
+                                              spypri == 2, "Medium (2)",
+                                              spypri == 3, "Low (3)",
+                                              ""
+                                              )
+  | extend
+      EventVendor = "SonicWall"
+    , EventProduct = "Firewall"
+    , sosDeviceModel = DeviceProduct
+    , DvcOs = "SonicOS"
+    , DvcOsVersion = EventProductVersion
+    , DvcIdType = "Other"
+    , Dvc = sosSerialNumber
+    , DvcDescription = DeviceProduct
+    , ASimMatchingHostname = "-"
+    , ASimMatchingIpAddr = "-"
+    , NetworkIcmpType = tostring(NetworkIcmpType)
+    , NetworkIcmpCode = toint(NetworkIcmpCode)
+    , NetworkBytes = tolong(coalesce(toint(ReceivedBytes), 0) + coalesce(toint(SentBytes), 0))
+    , FieldDeviceCustomNumber1Label = column_ifexists("FieldDeviceCustomNumber1Label", "") // FieldDeviceCustomNumberXLabel (cnXLabel=)
+    , FieldDeviceCustomNumber2Label = column_ifexists("FieldDeviceCustomNumber2Label", "") // FieldDeviceCustomNumberXLabel (cnXLabel=)
+    , DeviceCustomNumber1Label = column_ifexists("DeviceCustomNumber1Label", "") // DeviceCustomNumberXLabel (cnXLabel=)
+    , DeviceCustomNumber2Label = column_ifexists("DeviceCustomNumber2Label", "") // DeviceCustomNumberXLabel (cnXLabel=)
+    , DeviceCustomNumber3Label = column_ifexists("DeviceCustomNumber3Label", "") // Connection Duration in milliseconds (for Connnection Closed msgs)

```

Comment:
@jaimeesc 
DeviceCustomNumber3Label is already part of CommonSecurityLog table. I don't think checking for column_ifexists is required

Reply:
Okay. I removed those lines.

Reply:
@jaimeesc change not reflecting